CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+### Unreleased (Major)
+
+* Use an RSpec Formatter to track tests' execution times
+* Dropped support for Turnip < 2.0.0
```

Comment:
* List breaking changes first.
* Add suggestion by why switched to the RSpec formatter.

```suggestion
* __(breaking change)__ Dropped support for Turnip < 2.0.0
* Use an RSpec Formatter to track tests' execution times more accurately
```